handlers/handlers.go

```diff
@@ -183,6 +186,9 @@ func getCategoriesCountQuery(searchQuery url.Values) url.Values {
 func getCategoriesTypesCount(ctx context.Context, accessToken, collectionID string, query url.Values, searchC SearchClient, censusTopicCache *cache.Topic) ([]data.Category, []data.Topic, error) {
 	var options searchSDK.Options
 	options.Query = query
+	options.Headers = map[searchSDK.Header][]string{
+		searchSDK.Authorisation: {"Bearer " + accessToken},
```

Comment:
We need the collection id too.

Do we need to extend the list of available headers in the SDK package to include this? Or can we create our on key value pairs?

Reply:
You don't need to export the actual headers, just the Header struct.  You can write code like this:

``` 
options.Headers = map[searchSDK.Header][]string{
		searchSDK.Authorisation: {"Bearer " + accessToken},
		"header_name": { "header_value" },
```

The list of headers is just a nice to have.

Reply:
Yep, I thought as much. So we just need to figure out what the header is that holds the collection id and then add that to the map of headers

Reply:
Yeah I've updated both PRs now.

https://github.com/ONSdigital/dp-search-api/pull/184